tests/smoke/fake-amo-server.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+
+var http = require('http');
+
+const FAKE_REPLIES = [
+  {
+    url: 'http://localhost:8989/validation-results/',
+  },
+  {
+    guid: 'an-addon-guid',
+    active: true,
+    processed: true,
+    valid: true,
+    reviewed: true,
+    files: [{
+      signed: true,
+      download_url: 'http://localhost:8989/some-signed-file-1.2.3.xpi',
+    }],
+  },
+  {},
+];
```

Comment:
hmm, maybe we need to introduce some kind of spec file to generate this from [the source](http://addons-server.readthedocs.io/en/latest/topics/api/signing.html#get--api-v3-addons-[string-addon-id]-versions-[string-version]-(uploads-[string-upload-pk]-)) ? :) I'm just thinking out loud, may not be worth the effort.